firmware/library/L2_HAL/memory/sddriver.hpp

```diff
@@ -0,0 +1,1169 @@
+#pragma once
+
+#include <cstdint>
+#include <type_traits>
+#include "L1_Drivers/gpio.hpp"
+#include "L1_Drivers/ssp.hpp"
+#include "utility/log.hpp"
+
+// NOLINTNEXTLINE(readability-identifier-naming)
+// Relevant Datasheet:
+// https://www.sdcard.org/downloads/pls/pdf/index.php?p=Part1_Physical_Layer_Simplified_Specification_Ver6.00.jpg&f=Part1_Physical_Layer_Simplified_Specification_Ver6.00.pdf&e=EN_SS1
+class SdInterface
+{
+ public:
+  // @description     (see below)
```

Comment:
All doxygen documentation must use `///` to be recognized. Use `//` for comments that should not show up in the API doxygen page.

Reply:
Ok. Applied `///` only to comments that describe constants, member variables, and functions.